core/src/cats/cats.cc

```diff
@@ -3,7 +3,7 @@
 
    Copyright (C) 2011-2011 Free Software Foundation Europe e.V.
    Copyright (C) 2011-2016 Planets Communications B.V.
-   Copyright (C) 2013-2023 Bareos GmbH & Co. KG
+   Copyright (C) 2013-2024 Bareos GmbH & Co. KG
```

Comment:
If we don't have any change, this should also go away (after merging the fixups).

Reply:
Merged the fixups now.  Lets see what jenkins has to say.